facebook-commerce.php

```diff
@@ -178,9 +178,18 @@ class WC_Facebookcommerce_Integration extends WC_Integration {
 	 */
 	private $test_mode = false;
 
+	/** @var WC_Facebookcommerce_EventsTracker */
+	private $events_tracker;
+
+	/** @var WC_Facebookcommerce_MessengerChat*/
+	private $messenger_chat;
+
 	/** @var WC_Facebookcommerce */
 	private $facebook_for_woocommerce;
 
+	/** @var WC_Facebookcommerce_Background_Process */
+	private $background_processor;
```

Comment:
This change breaks the unit tests.

Please verify that this is not accessed outside the class in any place as it happens in the unit tests.
If everything is fine then probably `setAccessible` in UT will do the trick.